### DISCUSSION TITLE
subgraph edge attribute

### DISCUSSION THREAD

**Discussion by wubo2180** [pyg-team/pytorch_geometric#4218] [Q&A] [answered]
Hi,
When I call the subgraph function, the characteristics of my edge cannot be consistent with the dimension of the edge. Why is this
`data1.edge_index,data2.edge_attr=subgraph(idx_nondrop,data.edge_index,data.edge_attr,num_nodes=node_num)`

![image](https://user-images.githubusercontent.com/47293569/157180780-b3f4b3a8-bbd1-408d-b530-19153427359a.png)

**Comment by wubo2180** [comment, marked as answer]
Sorry, I am wrong written. It should be `        data1.edge_index,data1.edge_attr=subgraph(idx_nondrop,data.edge_index,data.edge_attr, num_nodes=node_num)`

**Reply by rusty1s** [reply]
Indeed. Does that resolve your issue?
